dbt/models/marts/users/_users__models.yml

```
@@ -97,11 +99,11 @@ models:
   #     data_tests:
   #       - not_null
     
-  - name: dim_user_course_activity
+  - name: dim_user_levels
     description: |
       This model is a log of all student- and teacher-course activity within a school year.   
 
-      Grain: user_id, school_year, course_name
+      Grain: user_id, level_id, activity_date
```

```suggestion
      Grain: user_id, level_id, script_id, activity_date
```
There are levels that appear in more than one script, so the grain should include both